topics/e2ee/index.md

```diff
@@ -0,0 +1,9 @@
+---
+display_name: E2EE
+aliases: end-to-end-encryption
+github_url: https://github.com/topics/e2ee
```

Comment:
@nholden What is the structure for the alias if we want to have E2EE point to End-to-End-Encryption?

I assume it's change `github_url` to point to `https://github.com/topics/end-to-end-encryption`

Then delete lines 5-9

?

Reply:
My understanding is that you can delete this file and keep `e2ee` as one of the `aliases` in the `end-to-end-encryption` index.

As an example: the [`cli` topic has `command-line-interface` as an alias](https://github.com/github/explore/blob/main/topics/cli/index.md). Both https://github.com/topics/cli and https://github.com/topics/command-line-interface pull metadata from the `cli` index file (there isn't a separate `command-line-interface` index file).

Reply:
CLI is a very helpful example. Stand by for the corrections.